tsds-grafana-handler/tsds_grafana_handler.py

```diff
@@ -333,7 +361,12 @@ def query():
                                 # Format the data received from tsds
                                 # to grafana compatible data
                                 datapoints = result[target_result['name']]
-                                target_result['datapoints'] = convert(datapoints)
+
+                                if isinstance(datapoints, list):
+                                        target_result['datapoints'] = convert(datapoints)
+                                else:
+                                        target_result['datapoints'] = [[datapoints, target_end]]
+
```

Comment:
Comments would be really good here - that's probably more of a general observation but here in particular is a good example of where we should be explaining what's going on here. It's obvious now, in 6mos it won't be.